extensions/authentication-jwt/src/services/user.service.ts

```diff
@@ -8,7 +8,7 @@ import {repository} from '@loopback/repository';
 import {HttpErrors} from '@loopback/rest';
 import {securityId, UserProfile} from '@loopback/security';
 import {compare} from 'bcryptjs';
-import {User} from '../models';
+import {User} from '../models/user.model';
```

Comment:
suspect the travis failure is caused by this line. 

Can you revert it to be 
```
import {User} from '../models';
```
?

Reply:
Reverted that, but still failing.

Reply:
See similar issues at https://github.com/strongloop/loopback-next/pull/5497

Reply:
Thanks @raymondfeng.